.changeset/itchy-ghosts-begin.md

```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-react': minor
+'@itwin/itwinui-css': minor
+---
+
+Added warning status for `ProgressRadial`/`ProgressLinear`
```

Comment:
I can add that in. Can you be more specific about using `patch` for `itwin-react`? I'm assuming patch is a tag or something similar I should add but there's nothing in the Contributing Docs I could find

Reply:
Oops, upon re-reading the code, this changeset was actually not needed 😅.

Changesets are only needed for user facing changes. But since this icon change was in the ladle stories (demos) and not part of the package, a changeset for it would not be needed.

So, https://github.com/iTwin/iTwinUI/pull/1813/files/0154705e15fbcb7512a9b0d79946b5762d565ddc..30331c7ace05ffbb97eaac7a588b7771ef5448ac can be undone.

Reply:
so no change set at all? or just stick to the original change set?

Reply:
[`5aacbbc` (#1813)](https://github.com/iTwin/iTwinUI/pull/1813/commits/5aacbbc4cc3608275aab48a711072ff3a7b82f4c) now addresses this 👍.

Reply:
ahh ok yeah I undid it